test/e2e/0xTrade.test.ts

```diff
@@ -0,0 +1,192 @@
+import ERC20 from "@openzeppelin/contracts/build/contracts/ERC20PresetMinterPauser.json";
```

Comment:
Is there something special about the `PresetMinterPauser` we are explicitly using that we couldn't just use regular `mintableERC20` for? Or even just ERC20?

Reply:
This is one of the OpenZeppelin "preset" contracts that implements `ERC20` and a `mint` function useful for e2e tests.

> st use regular mintableERC20 for?

When I looked, I didn't see such a preset (but I may have missed something).

> Or even just ERC20?

Its an `abstract` contract, so we can't create one directly, and does not allow `mint`-ing.